lib/android-tools-info.ts

```diff
@@ -177,6 +177,24 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		}).future<boolean>()();
 	}
 
+	public getPathToAdbFromAndroidHome(): IFuture<string> {
+		return (() => {
+			if(this.androidHome) {
+				let pathToAdb = path.join(this.androidHome, "platform-tools", "adb") + (this.$hostInfo.isWindows ? ".exe" : "");
```

Comment:
Not sure if it works without, it but it works with .exe and CI is green :) :100: